src/Simulation/Simulators.Tests/QuantumTestSuite/AssertProbMultiQubit.qs

```diff
@@ -8,6 +8,41 @@ namespace Microsoft.Quantum.Simulation.TestSuite {
     open Microsoft.Quantum.Simulation.TestSuite.Math;
     
     
+    internal operation flipToBasis (basis : Int[], qubits : Qubit[]) : Unit is Adj + Ctl {
```

Comment:
Is there a naming convention for internal operations, or should they use the same as public ones, e.g. `internal operation FlipToBasis`?

Reply:
I'm actually not sure. Previously this was `_flipToBasis` and that was acceptable, but it does make it seem like a local variable instead of a defined operation, so now that you've drawn my attention to it I'm inclined to change it!